projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,35 +4,31 @@
 
 ## 11.5-beta - 2022-10-25
 ### Enhancements
-- Add Contact Form child blocks to the Block Library [#26937]
-- Add default spacing attributes on all form variations. This achieves an easier way to select the form (parent) block while retaining wysiwyg capabilities: the user might choose to remove the spacing. [#26916]
-- Add Jetpack Search Free to my products. [#27007]
-- Add Jetpack Social to my products. [#26990]
-- Add post-purchase onboardings to Recommendation flows [#26484]
-- Change forms layout flex styles. Remove input's right border (used for spacing). Improve flex directives on 25, 50 and 75 wrappers. [#26914]
-- Display broken connections to user in editor [#25803]
-- Fix visual issues of the Product Price component in the Jetpack plugin [#27032]
-- Jetpack: Add unconnected site dashboard persistent connection widget [#26596]
-- Move Contact Forms child blocks to a new category and remove some Core blocks from the child blocks list [#26896]
-- Remove duplicated contact form settings from the Contact Form block's toolbar in favor of the sidebar [#26911]
+- Form block: add Contact Form child blocks to the Block Library. [#26937]
+- Form block: add default spacing attributes on all form variations. [#26916]
```

Comment:
I think these two need to move below the Dashboard entries (just so everything is in order of the initial capital letter.